src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+          >
+        </label>
+
+        <label class="form_field">
+          Name:
+          <input type="text" name="first_name">
+        </label>
+
+        <label class="form_field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form_field">
+          Full date of birth:
+          <input type="date" name="date_of_birth">
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <label class="form_field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="5"
+            maxlength="15"
+            class="no_margin"
+          >
+        </label>
+
+      </fieldset>
+
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form_field">
+          Do you love cats?
+
+          <label>
+            <input type="radio" name="options">
+            Yes
+          </label>
+
+          <label>
+            <input type="radio" name="options">
+            No
+          </label>
+        </div>
+
+        <label class="form_field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+
+        <div class="form_field">
+          <label for="time">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="time"
+            step="2"
+            id="time"
+          >
+        </div>
+
+        <label class="form_field">
+          What are your favorite brand of cars?
+          <select
+          name="cars"
+          id="cars"
+          multiple
+        >
```

Comment:
```suggestion
          <select
            name="cars"
            id="cars"
            multiple
          >
```
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)